packages/playground/src/pages/DropinUMD/session.js

```diff
@@ -39,24 +39,29 @@ export async function initSession() {
         }
     });
 
-    const dropin = new Dropin({
-        core: checkout,
-        instantPaymentTypes: ['googlepay'],
-        paymentMethodsConfiguration: {
-            paywithgoogle: {
-                buttonType: 'plain'
-            },
-            card: {
-                hasHolderName: true,
-                holderNameRequired: true,
-                holderName: 'J. Smith',
-                positionHolderNameOnTop: true,
-
-                // billingAddress config:
-                billingAddressRequired: true,
-                billingAddressMode: 'partial'
-            }
-        }
+    createComponent('card', {
```

Comment:
Is this supposed to be here? It looks like you were just testing something... and commented out the `new Dropin()` code